### ISSUE TITLE
Filtering pretraining data for Image Infilling

### ISSUE THREAD

**Issue by varadgunjal** [OFA-Sys/OFA#287]
Appendix A.1 states "We carefully filter our pretraining data and exclude images that appear in the validation and test sets
of downstream tasks to avoid data leakage." Is it possible to share more details about this filtration process? Particularly for the Image Infilling task, it looks like the total size of the 3 datasets is >100M [OpenImages (9M) + ImageNet (14M) + YFCC100M (99M)] whereas the table lists OFA as only using ~36M images. So I was curious as to how so much of the data was left out.

**Comment by logicwong** [comment]
@varadgunjal We only use a small subset of YFCC100M (14M). Check [here](https://github.com/OFA-Sys/OFA/blob/main/datasets.md) for the details of the pretraining datasets.

**Comment by varadgunjal** [comment]
For OpenImages, are the validation & test splits excluded?

**Comment by logicwong** [comment]
@varadgunjal Yes, we exclude the validation & test splits, and only use the train split. https://github.com/cvdfoundation/open-images-dataset#download-full-dataset-with-google-storage-transfer

![D3D32032-DEDE-4888-BB3F-A7B349AB30FA](https://user-images.githubusercontent.com/36780733/210922216-3628ce1f-7dd0-4436-a16c-0d2fb0c98279.png)
